index.html

```diff
@@ -0,0 +1,186 @@
+        
+       
+            <div class="project3-img">
+            </div>
+            <div class="project-des">
+              <h2 class="sub-title">Project name goes here</h2>
+              <ul>
+                <li>HTML/CSS</li>
+                <li>Ruby on Rails</li>
+                <li>Javascript</li>
+              </ul>
+              <div class="btn-project">
+              <button>See this  project   <i class="fa fa-long-arrow-right"></i>  </button> 
+              </div> 
+              </div>
+
+
+            <div class="project4-img">
+            </div>
+            <div class="project-des">
+              <h2 class="sub-title">Project name goes here</h2>
+              <ul>
+                <li>HTML/CSS</li>
+                <li>Ruby on Rails</li>
+                <li>Javascript</li>
+              </ul>
+              <div class="btn-project">
+               <button>See this  project   <i class="fa fa-long-arrow-right"></i>  </button>
+              
+              </div> 
+            </div>
+
+
+              <div class="project5-img">
+              </div>
+              <div class="project-des">
+                <h2 class="sub-title">Project name goes here</h2>
+                <ul>
+                   <li>HTML/CSS</li>
+                   <li>Ruby on Rails</li>
+                   <li>Javascript</li>
+                </ul>
+                <div class="btn-project">
+                 <button>See this  project   <i class="fa fa-long-arrow-right"></i>  </button>
+                </div> 
+              </div>
+
+                <div class="project6-img">
+                </div>
+                <div class="project-des">
+                  <h2 class="sub-title">Project name goes here</h2>
+                  <ul>
+                    <li>HTML/CSS</li>
+                    <li>Ruby on Rails</li>
+                    <li>Javascript</li>
+                  </ul>
+                  <div class="btn-project">
+                   <button>See this  project   <i class="fa fa-long-arrow-right"></i>  </button>
+                  </div> 
+                </div>
+          </div>
+ </section>
+
+ <section class="aboutme">
+    <h2 class="aboutme-title">
+       About
+    </h2>
+    <p class="aboutme-p">
+      I can help you build a product , feature or website Look through some of my work 
+      and experience! If you like what you see 
+      and have a project you need coded, don’t hesitate to contact me.
+    </p>
+    <button class="aboutme-btn">
+       <span> Get my  CV</span> <img src="./images/ic_download.png" alt="ic_download">
+    </button>
+    <div class="aboutme-cards">
+      <div class="aboutme-techs">
+        <h3 class="aboutme-techs-title">Languages</h3>
+        <ul class="techs">
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+            <li>CSS</li>
+        </ul>
+        <img class= "logo-corner" src="./images/ic_languages_mob.png" alt="languages">
+      </div>
```

Comment:
- [ ] Kindly use this as a `background-image` instead of an actual image. Decoration images are always applied as background images while `img` tags are used if the image is part of the content such as a logo or diagram or person. You can read this [article](https://www.w3docs.com/snippets/html/when-to-use-html-img-tag-and-css-background-image-property.html) to have more insight. You can pay attention to the snippet pasted below. 👍 😎 

![image](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/77106962/bbd5ad96-f979-4247-9eb9-55c70f185915)

#### Kindly apply this to all images you used in the technology categories for the **_About Section_**. 👍